### PR TITLE
Improve Custom Fields API documentation

### PR DESCRIPTION
**Pull request by KateVomBerg**

### CODE REVIEW

File: api-docs/custom-field-intro.md [hunk 1]
```diff
@@ -3,32 +3,58 @@ title: Custom Fields API - Introduction
...
+With our Custom Fields offering, Trackunit Iris can host all the data that you need. Custom Fields will enable you to capture and organize additional metadata that holds significant contextual value, thereby facilitating more robust data categorization and retrieval processes for you and your end-customers.
 
-Custom fields provide a way to define new fields in the Trackunit data model. Allowing to extend and customize Trackunit Manager. Currently we support **extending the data model of assets, accounts, groups and sites with new fields**. Apart from defining your own custom fields we also provide a number of standard fields for common metadata.
+# Additional standard fields for commom metadata
+Trackunit provides a number a number of standard fields for common metadata, that can be used by all customers to extend the available data model. These fields can exists across different domains of Trackunit's data model and currently can extend assets, accounts, groups or sites.
```
Comment (tandrup) [inline api-docs/custom-field-intro.md:9]: Small typo:
```suggestion
Trackunit provides a number of standard fields for common metadata, that can be used by all customers to extend the available data model. These fields can exists across different domains of Trackunit's data model and currently can extend assets, accounts, groups or sites.
```

Comment (tandrup) [inline api-docs/custom-field-intro.md:9]: Regarding the part about fields being available across assets, accounts, groups or sites. Right now we only have standard fields on asset. Don't know if it makes to list them out.

Comment (tandrup) [inline api-docs/custom-field-intro.md:9]: Btw. We have also added customers and rental contracts recently
image

Comment (KateVomBerg) [reply api-docs/custom-field-intro.md]: Nice! I have added the two new extendable domains when talking about true custom fields ✨

File: api-docs/custom-field-intro.md [hunk 2]
```diff
@@ -3,32 +3,58 @@ title: Custom Fields API - Introduction
...
+| Rated electric motor power | Power output of the electric motor in kilo-Watts | text (kW) | asset |
+| Total rated battery capacity | Total rated capacity summed (only if batteries in parallel) over all traction batteries, in ampere-hours. | text (Ah) | asset |
+| Total rated battery power capacity | Total rated energy capacity summed over all traction batteries, in kilo-watt-hours  | text (kWh) | asset |
+| Warranty end date | The end date of the asset's warranty. | date | asset |
+
+## How to set values on standard fields
+
+1. Use the [Custom Fields Values API](/reference/custom-fields-get-values) to get all available fields.
+- Specify the `entityId` for your query, which is an assetId for any fields listed above.
+- Specify which System Of Measurement you would like to use (SI or US customary)
```
Comment (tandrup) [inline api-docs/custom-field-intro.md:30]: I see that the docs says System of measurement is required. But that is actually a mistake in the docs. @jbd-trackunit will fix the docs.
It is really only relevant for the fuel tank capacity right now to select gallons vs liter.

**Review by tandrup — APPROVED** [review]
Should we do a call out on the asset api, site api, etc. like "Did you know that with IrisX you can extend the data model blah blah blah"

File: api-docs/custom-field-intro.md (hunk 2 above)
Comment (KateVomBerg) [reply api-docs/custom-field-intro.md]: Great 👍 I have removed the mention from the docs.

**Pull request merged**